assets/src/edit-story/components/helpCenter/useHelpCenter.js

```diff
@@ -86,12 +92,39 @@ const initial = {
     }),
     toggle: () => ({ isOpen }) => ({ isOpen: !isOpen }),
     close: () => () => ({ isOpen: false }),
+    fetchReadStart: () => () => ({ loading: true }),
+    fetchReadSuccess: (payload) => () => ({
+      ...payload,
+      loading: false,
+    }),
+    fetchReadError: () => () => ({
+      fetchError: true,
+      loading: false,
+    }),
+    read: (key) => ({ read }) => ({
+      // optimistically update the notification as "read"
+      read: read.indexOf(key) === -1 ? read.concat(key) : read,
+      updating: true,
+    }),
+    readSuccess: () => () => ({
+      updating: false,
+    }),
+    readError: (key) => ({ read }) => ({
+      updating: false,
+      readError: true,
+      read: read.filter((tipKey) => tipKey !== key),
+    }),
```

Comment:
I don't think we need these actions right now. If a user reads a tip and our endpoint is down, it should still show that they read the tip, but should just not be persisted between refreshes

Reply:
Maybe the error is still nice to have so we can eventually show some alert or toast that says that read tips arent being persisted:
```
    persistingReadTipsError: () => () => ({
      readError: true,
    }),
```